src/index.ts

```diff
@@ -181,21 +171,41 @@ const fetchOmnivore = async (inBackground = false) => {
               return compareHighlightsInFile(a, b)
             }
           })
+        const highlightBatch: IBatchBlock[] =
+          article.highlights?.map((it) => {
+            const noteChild = it.annotation
+              ? { content: it.annotation }
+              : undefined
+            return {
+              content: `>> ${markdownEscape(
```

Comment:
Hey @jacksonh, I didn't find a method from Logseq docs to do this so I escape the markdown special characters manually.

Only the `quote` is being escaped and the others are not since the `title` is wrapped in a `[]` and the `annotation` is fine to be a markdown text.